src/toniarts/openkeeper/gui/Cursor.java

```diff
@@ -19,17 +19,18 @@
 import com.jme3.asset.AssetManager;
 import com.jme3.cursors.plugins.JmeCursor;
 import com.jme3.texture.Image;
+import com.jme3.texture.Image.Format;
 import com.jme3.texture.Texture;
 import com.jme3.util.BufferUtils;
 import java.io.File;
 import java.nio.ByteBuffer;
 import java.nio.IntBuffer;
-import java.util.regex.Matcher;
 import toniarts.openkeeper.tools.convert.AssetsConverter;
+import toniarts.openkeeper.tools.convert.ConversionUtils;
 
 public class Cursor extends JmeCursor {
 
-    public int delay = 30;
+    public final int delay = 30;
```

Comment:
Oooh, my bad, this should be a constant (final static)? No need to change it though, just spotted here :)